src/schema/me/__tests__/bidder_position.test.js

```diff
@@ -0,0 +1,44 @@
+import { runAuthenticatedQuery } from "test/utils"
+
+describe("BidderPosition", () => {
+  it("returns processed_at", () => {
+    const rootValue = {
+      bidderPositionLoader: sinon.stub().returns(
```

Comment:
You shouldn't need the stub here since you can just return a function like `bidderPositionLoader: () => Promise.resolve...`